Lib/asyncio/windows_events.py

```diff
@@ -512,6 +512,26 @@ def finish_recv(trans, key, ov):
 
         return self._register(ov, conn, finish_recv)
 
+    def recvfrom_into(self, conn, buf, flags=0):
+        self._register_with_iocp(conn)
+        ov = _overlapped.Overlapped(NULL)
+        try:
+            ov.WSARecvFromInto(conn.fileno(), buf, flags)
+        except BrokenPipeError:
+            return self._result((b'', None))
```

Comment:
Should it set `(0, None)` instead?

Reply:
Fixed. I'm not sure what would cause BrokenPipeError to be raised with a datagram socket, so I'm not sure this should even be here.

Reply:
I don't know either.  MSDN is not very descriptive here.